app/assets/stylesheets/patterns/_patterns.scss

```diff
@@ -1,3 +1,4 @@
-@import "titles";
 @import "buttons";
+@import "font";
```

Comment:
Rename to use plural variation for the file name?

Reply:
Done with fixup in `0807dc2`